### DISCUSSION TITLE
Audio Gridder does not show up in pro tools ultimate trail legit 2021.0.6

### DISCUSSION THREAD

**Discussion by Jarrodsz** [apohl79/audiogridder#852] [Help]
<img width="627" alt="Screenshot 2021-12-14 at 11 43 40" src="https://user-images.githubusercontent.com/90145421/145983391-718c7f5e-8c91-4d93-9f5f-f02b13902cb5.png">

They are correctly in my Avid folder
<img width="767" alt="Screenshot 2021-12-14 at 11 44 05" src="https://user-images.githubusercontent.com/90145421/145983464-42f72474-c2a0-40d7-b788-0566fca8bdc1.png">

How to I get pro tools to load them correctly?

**Comment by Jarrodsz** [comment]
Edit:
Im running om M1 MAX macbook pro 2021.
Tried to install the INTEL version also but to no avail it still not showing up. Using beta version latest.
Before I had the M1 version installed.

MacOS Montery 12.1

**Comment by Jarrodsz** [comment]
anyone could please confirm that this should work? Im on pro tools 2021.12 now and the plugin WILL NOT SHOW up in pro tools. Tried mono.stero.instrument tracks nothing works... how to resolve?

**Reply by Watto5150** [reply]
I got the Beta 1.2 version working fine so far But im on PC
The folder hierarchy is all wrong with the beta 1.2 version aax plugin the aax just goes to plugins folder, not in folders like the 1.1 version so do this
Install version 1.1 then go to c drive/program files/common files/Avid/audio/plug-ins
And copy the 3 Audiogridder folders and paste them to a folder on your desktop
then uninstall version 1.1 and install beta v1.2
then go back to c drive/program files/common files/Avid/audio/plug-ins
and cut the aax plugin and replace it to the one in folders on your desktop/folder you created/Audiogridder.aaxplugin/contents/x64 paste and over ride original file
and do the the same with instrument and midi aax
then copy the 3 folders back to the plugins folder
do a restart then it works fine
The server needs VST plugins installed on it it dosent scan AAX kind of works similar to Blue Cats Patchworks as a vst wrapper
Tested it with waves V13 plugins so far what i Found so far VST presets not visible But the VST3 presets work 
Noticed minimal latency on default settings with 5 Plugins on the chain
Haven't fully tested it yet but got it working for an hour
